### DISCUSSION TITLE
Is the sample batect.yml provided here correct?

### DISCUSSION THREAD

**Discussion by Bidaya0** [batect/batect#1323] [General]
https://batect.dev/docs/reference/config/containers#build_target

please review this https://github.com/batect/batect.dev/pull/1033

btw: I am making a cli program that converts from docker-compose.yml to batect.yml and need to know every field inside batect.yml

**Comment by charleskorn** [comment]
Good catch, thanks @Bidaya0!

> btw: I am making a cli program that converts from docker-compose.yml to batect.yml and need to know every field inside batect.yml

Have you seen @ineffyble's [batectify](https://github.com/ineffyble/batectify)? There might be something there that you can borrow / use as inspiration.

**Reply by Bidaya0** [reply]
> 

Thanks， I will take a look at it .

**Reply by Bidaya0** [reply]
> Good catch, thanks @Bidaya0!
> 
> > btw: I am making a cli program that converts from docker-compose.yml to batect.yml and need to know every field inside batect.yml
> 
> Have you seen @ineffyble's [batectify](https://github.com/ineffyble/batectify)? There might be something there that you can borrow / use as inspiration.

Thank you for your prompt reply.
Please take another look at this pr https://github.com/batect/batect.dev/pull/1035.
Also found out when I implemented my converter.
Thanks to you, a most original version has been released and can cover my current project using batect.
